### ISSUE TITLE
Have AccessControlRegistry implement initializeRole() and have contracts with roles initialize them in constructors

### ISSUE THREAD

**Issue by bbenligiray** [api3dao/airnode-protocol-v1#33]
Ideally, contracts that use roles from AccessControlRegistry should initialize them at constructor. This is currently not possible because the only function to initialize a role also requires the sender to be able to grant it, which means calling it in the constructor will revert. If we implement a function that only initializes a role but not grants it in AccessControlRegistry, we can implement this. This requires AccessControlRegistry to be redeployed, which is not ideal (but not out of the question either).

**Comment by bbenligiray** [comment]
Multicall that AccessControlRegistry inherits [has been updated](https://github.com/api3dao/airnode-protocol-v1/pull/32), which means AccessControlRegistry will be redeployed anyway. This can now be implemented too.

**Comment by bbenligiray** [comment]
After implementing this, I remembered why AccessControlRegistry was implemented the way it is. When we let anyone to initialize a role, users can initialize roles for managers and Airnodes, cluttering up the roles that depend on them. Although these can be filtered, this is difficult to do correctly. Also, when an Airnode account interacts with a ...WithAirnode contract, they will still need to create their roles explicitly. Therefore, addressing this issue only affects the ...WithManager contracts and create inconsistency in user flow. If having to explicitly create roles is unacceptable user flow, we can't expect Airnode accounts to go through it either (and we don't think that this is the case, which means we don't need to try to improve the manager-based flow either).
Finally, AccessControlRegistry doesn't need SelfMulticall.sol (i.e., tryMulticall()) because its functions are implemented in a way that avoid reverts (for example when one initializes an already initialized role, the function does nothing instead of reverting).
